README.md

```diff
@@ -1 +1,173 @@
-# Portfolio
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+  # Porkalai Karthika
+  <br/>
+
+  <h3><b>My portfolio</b></h3>
+
+</div>
+
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage) 
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+
+
+# 📖 My Portfolio setup<a name="about-project"></a>
+
+> Created my profolio project using HTML and CSS
+
+
+
+## 🛠 Built With  <a name="built-with"></a>
+1-HTML</br>
+2-CSS</br>
+3-LINTERS</br>
+### Tech Stack <a name="tech-stack"></a>
+
+> 
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://index.html/">HTML</a></li>
+  </ul>
+</details>
+
+
+
+
+
+### Key Features <a name="key-features"></a>
+
+>
+- **Added menu** [My logo and menu icon added] 
+- **Styled using flex** [This page is styled using css flex]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+-Text editor</br>
+-Web browser
+
+### Setup
+
+Clone this repository to your desired folder:
+
+
+Run command: https://github.com/Porkalaikarthika/Portfolio.git
+### Install
+
+Install this project with:
+
+
+  
+  npm install
+
+
+### Usage
+
+To run the project, execute the following command:
+
+Open the index.html file
+
+
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+## 👥 Authors <a name="authors"></a>
+
+
+
+👤 Porkalai karthika
+
+- GitHub: [@githubporkalaikarthika](https://github.com/Porkalaikarthika)
+- Twitter: [@twitterkarthika](https://twitter.com/PorkalaiKarthi)
+- LinkedIn: [LinkedIn](linkedin.com/in/porkalai-karthika/)
```

Comment:
- **[OPTIONAL]** The link to your LinkedIn profile is broken, please 🙏🏽 fix it for professional documentation.